### ISSUE TITLE
Unnecesary warning regarding clearing.

### ISSUE THREAD

**Issue by reduz** [KhronosGroup/Vulkan-ValidationLayers#1307]
I am trying to render to a region of a framebuffer, and only clearing this region of the framebuffer, as I intend the rest of the content to remain unmodified. As clearing a portion is not possible on Vulkan (I understand this is done for performance reasons, specially on mobile or for hzb, but for my specific use case I just intend to update an atlas with something simple, so it's fine), I use LOAD_OP_KEEP and then emit a clear option (vkCmdClearAttachments) on the region I intend to draw to (via vkCmdSetViewport).

However, I get this warning:
```
WARNING : PERFORMANCE - Message Id Number: 0 | Message Id Name: UNASSIGNED-CoreValidation-DrawState-ClearCmdBeforeDraw
	vkCmdClearAttachments() issued on command buffer object 0x6596ca0 prior to any Draw Cmds. It is recommended you use RenderPass LOAD_OP_CLEAR on Attachments prior to any Draw.

	Objects - 1
		Object[0] - VK_OBJECT_TYPE_COMMAND_BUFFER, Handle 0x6596ca0
```

I understand the spirit of it, but my use case is entirely valid. Maybe the validation layer should detect that I am only clearing a small region and not the entire framebuffer, hence not show this?

**Comment by mark-lunarg** [comment]
@reduz, we're in the (slow) process of moving warnings out of the default operation of the validation layers, and into an opt-in validation state (Best Practices).  I am adding this to the Best Practices tracking issue #24, and it will be added there and removed from corechecks.  @camden-lunarg will address this as part of that issue.  Thanks!
